src/sentry/integrations/slack/message_builder/issues.py

```diff
@@ -277,7 +277,7 @@ def build(self) -> SlackBody:
             actions=payload_actions,
             callback_id=json.dumps({"issue": self.group.id}),
             color=color,
-            fallback=f"[{project.slug}] {obj.title}",
+            fallback=self.build_fallback_text(obj, project.slug),
```

Comment:
Could there be cases when `self.event` and `self.group` are both None?

Reply:
If the typing is to be trusted there will always at least be a group